io.github.Strozer2002.sparrow.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.Strozer2002.sparrow
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R bin/ /app/sparrow
+      - mkdir -p /app/sparrow
```

Comment:
there is something wrong in that command order. Either you don't need the mkdir or the content is copied in the wrong place.

Reply:
also why not building from source?

Reply:
It's flutter apparently

Reply:
Sorry i dont know how. I take this yml from youtube example

Reply:
You can write example?

Reply:
Yes? its flutter

Reply:
> You can write example?

Move `mkdir -p /app/sparrow` before `cp`, change `cp` to `cp -R sparrow-wallet/bin`

Reply:
You write something wrong 
![image](https://github.com/flathub/flathub/assets/80827297/b4b76f65-b909-4f70-af79-37d318f5ea07)

Reply:
You write something wrong
![image](https://github.com/flathub/flathub/assets/80827297/74434993-c59d-4d79-bdc9-5ff26e594355)

Reply:
the last two suggestions will fix the issue

Reply:
> You write something wrong 

The next two suggestions will fix the issue